src/main/java/net/openhft/chronicle/bytes/internal/migration/HashCodeEqualsUtil.java

```diff
@@ -5,17 +5,15 @@
 import net.openhft.chronicle.bytes.internal.BytesInternal;
 import net.openhft.chronicle.core.io.ReferenceOwner;
 
-import static net.openhft.chronicle.bytes.internal.ReferenceCountedUtil.throwExceptionIfReleased;
-
 public final class HashCodeEqualsUtil {
 
     private HashCodeEqualsUtil() {
     }
 
     // Comparing and reserving at the same time
-    // Declared private at the moment so not public
-    private static boolean contentEquals(final BytesStore<?, ?> bytes,
-                                        final Object other) {
+    // Declared package private at the moment so not public
+    static boolean contentEquals(final BytesStore<?, ?> bytes,
```

Comment:
Why? Are there plans to use it?

Reply:
Why do we need to expose it in package private fashion if it wasn't previously?

Reply:
I will remove it

Reply:
It will be slow when tracing is enabled as it is by default in this module. It shouldn't be slow when this is off.